server/models/Column.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require("mongoose");
+
+const columnSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true,
+    unique: true
```

Comment:
With this kind of application, we don't want to set the title unique. In the database the columns collection cannot have more than one "Done" this is a poor user experience and won't be very practical, please remove this property. Thanks!

Reply:
Essentially, only one user ever can have "Done" and it's not per-user basis.